.gitignore

```diff
@@ -102,3 +102,6 @@ ENV/
 
 # Autogenerated files
 *pb2*.py
+
+# K8s secrets
+*.secrets.yaml
```

Comment:
Can we move this to our private Kubernetes repo (we're adding secrets it in the `data-clusterconfig/secrets` folder)? 

I think we could even remove this secret as it's a combination of others! 😊

Reply:
The reason I chose to rather use separate secrets it to be able to keep them contained for this application, since the pipeline is meant to be open source and deployable as a standalone application. 

An additional benefit of not using shared key is that the AWS IAM user keys can be scoped to only need access to resources it needs (the S3 bucket for loading data into Redshift)

For our own Kubernetes cluster, I'm happy to perhaps rely on the private repo, perhaps we could just duplicate the secrets there?

Reply:
> AWS IAM user keys can be scoped to only need access to resources it needs (the S3 bucket for loading data into Redshift)

Awesome! This feels safer. 🙌  

> perhaps we could just duplicate the secrets there?

I think we could create a `secret.yaml` file to share openly with the required secrets and dummy values. The deployment file then could point to our own private secrets. 

This way we'll share the entire set of required YAML files (without ignoring the secrets one) and have another private ourselves. 

How would that sound?